src/config/config.ts

```diff
@@ -110,7 +110,7 @@ export class Configuration {
 
   blockchain = {
     minFeeRate: 0.00001,
-    minFeeBuffer: 1,
+    minFeeBuffer: 0.1,
```

Comment:
was discussed in a group chat, that we should reduce this and have the same "guard" as the LW. Ours is working slightly differently. But otherwise we would need to send 1+ DFI UTXOs to vault owner addresses to be able to generate raw-txs. With this change, we can just send 0.2 batches or similar.